CMakeLists.txt

```diff
@@ -23,6 +23,17 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake;${CMAKE_MODULE_PATH}")
 
 set(XEUS_CPP_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/include)
 
+# Install xeus-cpp tag files
+set(XEUS_CPP_DATA_DIR "share/xeus-cpp" CACHE STRING "xeus-cpp data directory")
+set(XCPP_TAGFILES_DIR ${CMAKE_CURRENT_SOURCE_DIR}/share/xeus-cpp/tagfiles)
+install(DIRECTORY ${XCPP_TAGFILES_DIR}
+        DESTINATION ${XEUS_CPP_DATA_DIR})
+
+set(XEUS_CPP_CONF_DIR "etc/xeus-cpp"   CACHE STRING "xeus-cpp configuration directory")
+set(XCPP_TAGCONFS_DIR ${CMAKE_CURRENT_SOURCE_DIR}/etc/xeus-cpp/tags.d)
+install(DIRECTORY ${XCPP_TAGCONFS_DIR}
+        DESTINATION ${XEUS_CPP_CONF_DIR})
```

Comment:
@tharun571 Probably just set the variables here, and move the install part to the install section in the cmakelists.txt file.